src/containers/my-cooperations/my-cooperations-details/MyCooperationsDetails.tsx

```diff
@@ -105,9 +145,21 @@ const MyCooperationsDetails = () => {
     displayedUser.photo &&
     createUrlPath(import.meta.env.VITE_APP_IMG_USER_URL, displayedUser.photo)
 
-  const cooperationCompletion = userRole === UserRoleEnum.Tutor && (
-    <CooperationCompletion />
-  )
+  const handleCooperationStatusUpdate = async () => {
+    const confirmed = await checkConfirmation({
+      title: 'titles.confirmCooperationClosing',
```

Comment:
You forget to use `t` from `useTranslation` here

Reply:
fixed